scripts/uberenv/packages/axom/package.py

```diff
@@ -507,13 +507,16 @@ def hostconfig(self, spec, prefix):
 
                 if not spec.satisfies('cuda_arch=none'):
                     cuda_arch = spec.variants['cuda_arch'].value
-                    axom_arch = 'sm_{0}'.format(cuda_arch[0])
-                    cfg.write(cmake_cache_entry("AXOM_CUDA_ARCH", axom_arch))
-                    cudaflags += "-arch ${AXOM_CUDA_ARCH} "
+                    cfg.write(cmake_cache_entry("CMAKE_CUDA_ARCHITECTURES", cuda_arch))
```

Comment:
Is this a proper CMake variable now?

Reply:
Yes... required for CMake 3.18, otherwise it errors out.